api/proto/scheduler/controller.proto

```diff
@@ -14,8 +14,10 @@ message ExecutionContext {
 
 message ActionRequest {
     uint32 action_id = 1;
+    
     ExecutionContext context = 2;
-    repeated string commands = 3;
+    string repo_url = 3;
+    repeated string commands = 4;
```

Comment:
same here. Also i am wondering ; is the line break between fields 1 and 2 a typo?